types/globalize/dist/globalize/date.d.ts

```diff
@@ -32,6 +32,11 @@ declare module "../globalize" {
          */
         timeZone?: string;
     }
+    type DateFormatPartTypes = "day" | "dayperiod" | "era" | "hour" | "literal" | "minute" | "month" | "second" | "zone" | "weekday" | "year";
```

Comment:
Out of curiosity, what's the `literal` part?

Reply:
Any text in the formatted date that is not part of the date. e.g. Separators.

```js
> Globalize.locale('en-CA')
> Globalize.formatDateToParts(new Date, {datetime: 'full'});
[
  { type: 'weekday', value: 'Tuesday' },
  { type: 'literal', value: ', ' },
  { type: 'month', value: 'May' },
  { type: 'literal', value: ' ' },
  { type: 'day', value: '26' },
  { type: 'literal', value: ', ' },
  { type: 'year', value: '2020' },
  { type: 'literal', value: ' at ' },
  { type: 'hour', value: '6' },
  { type: 'literal', value: ':' },
  { type: 'minute', value: '48' },
  { type: 'literal', value: ':' },
  { type: 'second', value: '29' },
  { type: 'literal', value: ' ' },
  { type: 'dayperiod', value: 'p.m.' },
  { type: 'literal', value: ' ' },
  { type: 'zone', value: 'GMT-07:00' }
]
```

Some locales are also *"fun"* in including a literal "h" character in their time string.
```js
> Globalize.locale('fr-CA');
> Globalize.formatDateToParts(new Date, {time: 'short'});
[
  { type: 'hour', value: '18' },
  { type: 'literal', value: ' h ' },
  { type: 'minute', value: '49' }
]
```